### PR TITLE
Try to fix socket error

### PR DESCRIPTION
**Pull request by anmekin**

### CODE REVIEW

File: omaha_server/omaha_server/utils.py [hunk 1]
```diff
@@ -33,6 +34,10 @@ def emit(self, record):
             msg = msg.encode('utf-8')
         try:
             self.socket.sendto(msg, self.address)
+        except socket.error:
+                self.socket.close()
+                self.socket = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+                self.socket.send(msg)
```
Comment (kyakovenko) [inline omaha_server/omaha_server/utils.py]: It seems we should use the sendto method instead of send.

**Comment by yurtaev** [comment]
Why did you reimplement `SysLogHandler`?

**Comment by yurtaev** [comment]
Celery logs:

```
WARNING 2016-05-03 08:18:37,071 log 164 139910686178344 Traceback (most recent call last):
WARNING 2016-05-03 08:18:37,072 log 164 139910686178344 File "/srv/omaha/omaha_server/omaha_server/utils.py", line 35, in emit
WARNING 2016-05-03 08:18:37,072 log 164 139910686178344 self.socket.sendto(msg, self.address)
WARNING 2016-05-03 08:18:37,073 log 164 139910686178344 error: [Errno 88] Not a socket
WARNING 2016-05-03 08:18:37,074 log 164 139910686178344 Logged from file beat.py, line 202
```

**Comment by yurtaev** [comment]
Fixed in #179

**Pull request closed without merging**